src/graphiql-fetcher.ts

```diff
@@ -0,0 +1,32 @@
+import { SubscriptionClient } from './client';
+
+export const graphQLFetcher = (subscriptionsClient: SubscriptionClient, fallbackFetcher: Function) => {
+  let activeSubscriptionId: number | null = null;
+
+  return (graphQLParams: any) => {
+    if (subscriptionsClient && activeSubscriptionId !== null) {
+      subscriptionsClient.unsubscribe(activeSubscriptionId);
+    }
+
+    if (subscriptionsClient && graphQLParams.query.startsWith('subscription')) {
```

Comment:
GraphiQL exposes `string` to the fetcher, and not GraphQL parsed object (`DocumentNode`). So we have two options:
1. Add dependency for `graphql`, parse the string from GraphiQL and use it to check for subscriptions operations. 
2. Leave as is - maybe look for a substring instead only `startsWith`, or maybe other logic you think is better.

@NeoPhi @helfer what do you think is the preferred solution? or maybe I'm missing a third one?